go.mod

```diff
@@ -21,48 +21,48 @@ require (
 	github.com/contiv/ofnet v0.0.0-00010101000000-000000000000
 	github.com/coreos/go-iptables v0.4.5
 	github.com/elazarl/goproxy v0.0.0-20190911111923-ecfe977594f1 // indirect
-	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
-	github.com/go-openapi/spec v0.19.3
-	github.com/gogo/protobuf v1.3.1
-	github.com/golang/mock v1.4.4
-	github.com/golang/protobuf v1.3.2
-	github.com/google/uuid v1.1.1
+	github.com/go-openapi/spec v0.19.5
+	github.com/gogo/protobuf v1.3.2
+	github.com/golang/mock v1.5.0
+	github.com/golang/protobuf v1.4.3
+	github.com/google/uuid v1.1.2
 	github.com/kevinburke/ssh_config v0.0.0-20190725054713-01f96b0aa0cd
-	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/common v0.4.1
+	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus/common v0.10.0
 	github.com/rakelkar/gonetsh v0.0.0-20210226024844-dfffed138500
 	github.com/satori/go.uuid v1.2.0
-	github.com/sirupsen/logrus v1.4.2
+	github.com/sirupsen/logrus v1.7.0
 	github.com/spf13/afero v1.4.1
-	github.com/spf13/cobra v0.0.5
+	github.com/spf13/cobra v1.1.1
 	github.com/spf13/pflag v1.0.5
 	github.com/streamrail/concurrent-map v0.0.0-20160823150647-8bf1e9bacbf6 // indirect
 	github.com/stretchr/testify v1.6.1
 	github.com/ti-mo/conntrack v0.3.0
 	github.com/vishvananda/netlink v1.1.0
-	github.com/vmware/go-ipfix v0.4.7
-	golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a
-	golang.org/x/exp v0.0.0-20190312203227-4b39c73a6495
+	github.com/vmware/go-ipfix v0.4.8
```

Comment:
@tnqn Could you hold on a bit before merging? We moved to go-ipfix v0.5.0 tag as we had to include some changes for the last release. This tag is not valid anymore.
Will provide a new one soon.. @zyiou

Reply:
I think they can use v0.4.8 tag. 
We need to add klog from v1 -> v2 back again in v0.5.1 release. Currently Antrea main branch is using v0.4.7. After this PR gets merged, we can move go-ipfix to v0.5.1 in the PR that needs the latest change. @srikartati do you see any issue here?

Reply:
I was under the impression that Antrea main is using go-ipfix v0.5.0 tag already and we are downgrading. Yes you are right that we are still at v0.4.7, so it is definitely ok to move to v0.4.8.  Let's make sure we add the v2 log change in v0.5.1.

@tnqn There are no issues; please merge this PR.

Reply:
Thanks for confirming @zyiou @srikartati